src/sdk/base/publication.js

```diff
@@ -33,7 +33,8 @@ export class AudioPublicationSettings {
  */
 export class VideoPublicationSettings {
   // eslint-disable-next-line require-jsdoc
-  constructor(codec, resolution, frameRate, bitrate, keyFrameInterval, rid) {
+  constructor(codec, resolution, frameRate,
+      bitrate, keyFrameInterval, rid) {
```

Comment:
`rid` has the same issue as `id` in my previous comment. A settings is not tightly associated with a transceiver. It can be reused.

Reply:
Since `rid` is not introduced by this PR. Let's fix it in a separate change. A new issue #425 is created.